app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] After fixing the seed problem, now I'm unable to start the app with the command `./bin/dev` as required. It would be nice to add the command to our README file
![Screenshot from 2023-04-03 16-15-22](https://user-images.githubusercontent.com/46388113/229567500-f72545b3-ccf3-4b26-b741-f259cabd5dfb.png)
![Screenshot from 2023-04-03 16-17-25](https://user-images.githubusercontent.com/46388113/229567837-9943f83b-6891-4e3e-a4a9-b5db229f92cc.png)